crates/ruff_python_literal/src/format.rs

```diff
@@ -206,6 +236,8 @@ pub struct FormatSpec {
     precision: Option<usize>,
     // Ex) `f` in `'{:+f}'`
     format_type: Option<FormatType>,
+    // Ex) `x` and `y` in `'{:*{x},{y}b}'`
+    pub replacements: Vec<FormatPart>,
```

Comment:
I can switch this out for an access method

Reply:
+1, an access method that returns `&[FormatSpec]` makes sense to me.